provisioning-workflows/zOS/NetworkResources/README.md

```diff
@@ -0,0 +1,45 @@
+	
+# Prerequisites 
+
+ * IBM Cloud Provisioning and Management for z/OS is required.
+ 
+ 
+# Setup
+
+1. Download the workflow files to a z/OS system running IBM Cloud Provisioning and Management.
+2. Create a new template for the Network Resource Provisioning following the instructions in IBM Cloud Provisioning and Management.
+3. Associate the template to a tenant (user or group of users) who will be requesting network resources and create network resource pool.
+4. Work with network administrator to complete network resource pool in z/OSMF Network Configuration Assistance plugin identifying IP ranges, port ranges and SNA APPL ID ranges.
+5. Test the template. Thevworkflow provided in "Provision_Network_Resource.xml" will be driven to provision the requested network resource.
+6. Publish the template 
+
+
+# Provision Network Resource (Run the template)
+
+Provision Network Resource service provides capablity to provision network resources based on properties provided by the user. Following properties can be provided at the provisioning time. 
+
+* OBTAIN_DVIPA - Identifies whether dynamic VIPA should be obtained. Specify "YES" or "NO". If no value is specified, dynamic VIPA will not be provisioned.
+* OBTAIN_PORT - Identifies whether TCP/IP port should be obtained. Specify "YES" or "NO". If no value is specified, TCP/IP port will not be provisioned.
+* OBTAIN_APPLID - Identifies whether SNA Application ID (APPLID) should be obtained. Specify "YES" or "NO". If no value is specified, SNA APPLID will not be provisioned.
+* JOB_NAME - Identifies z/OS job that will be utilizing the allocated port. This property is required when TCP/IP port is requested. If job name is not known, "*" can be specified as value.
+* USAGE_TYPE - Optional property that identifies type of application (e.g. WLP, CICS or Node) for which port needs to be provisioned. When pool of port ranges defined in z/OSMF Network Configuration Assistant, network administrator can identify usage type for specific port ranges. USAGE_TYPE property must match with usage type defined in z/OSMF Network Configuration Assistant otherwise the port provisioning request will fail. When USAGE_TYPE property is not specified, port is provisioned from a port ranges pool that is defined with out usage type. 
+
+
+# Output - Provisioned Network Resources
+
+Once provisioning request completes, following properties are available in the provisioned instance based on the type of network resource requested at the provisioning time
+
+* IP_ADDRESS - This property provides the provisioned dynamic VIPA  when OBTAIN_DVIPA property was set at the provisioning time
```

Comment:
I think that the descriptions here could be used to better describe some of the properties in the workflow definition itself. This would also help users co-relate what they see in these instructions to the workflow definition

Reply:
Purpose of readme file is to provide additional info that is not in workflow and explain purpose of the service and input/output from service. But, I will see if the property explanation from read me can be applied to variable description in WF.